src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -144,6 +151,10 @@ internal static string GetRuntimeVersion()
             {
                 return FrameworkVersionHelper.GetFrameworkDescription();
             }
+            else if (IsBenchmarkTargetWasm || IsWasm())
```

Comment:
`IsBenchmarkTargetWasm` should be removed for the reasons explained above

```suggestion
            else if (IsWasm())
```